docs/reference/src/main/asciidoc/configure.asciidoc

```diff
@@ -1,6 +1,7 @@
 [[configure]]
 == Configuration
 
+[[weld-configuration]]
```

Comment:
The id of this section is currently autogenerated and has a value of `_weld_configuration`. We should probably keep this value so that for example existing links do not break.